src/scene_worker.js

```diff
@@ -31,19 +37,19 @@ if (Utils.isWorkerThread) {
     };
 
     // Starts a config refresh
-    SceneWorker.worker.updateConfig = function ({ tile_source, config }) {
-        SceneWorker.config = null;
```

Comment:
Where do these get reset now? They need to be cleared whenever `updateConfig` is called (it's basically a re-initialization step).